### PR TITLE
Bump KR memory version to 6.4

### LINKED ISSUE #259: KR 6.40 can't get CurrentWorldID from  `callOverlayHandler::getCombatants`
getting combatant list and i need using player's currentworldid. but `(...).combatants`  object array doesn't have CurrentWorldID (same in customlogline 261 CombatantMemory `Add`)

### PR DESCRIPTION
**Pull request by valarnin**
Might fix #259, needs tested by someone on the KR version.

### CODE REVIEW

**Review by quisquous — APPROVED** [review]

**Comment by valarnin** [comment]
(waiting on confirmation from someone that has the KR client that this resolves the issue before merging)

**Comment by quisquous** [comment]
I think this is correct to merge regardless, for what it's worth.

**Comment by valarnin** [comment]
I want to avoid merging this and having people think that it resolves the issue fully when further changes may need to be made.

**Pull request merged**
